include/cute/atom/copy_traits_xe.hpp

```diff
@@ -364,8 +369,6 @@ struct XE_2D_LD_Unpack {
     static_assert(is_rmem<TD>::value);
     static_assert(size(SLayout{}) * dtype_bits == size<1>(typename Traits_LD_t::SrcLayout{}),
                   "Src tensor size does not match copy atom size.");
-    static_assert(size(DLayout{}) * dtype_bits == size<1>(typename Traits_LD_t::DstLayout{}),
-                  "Dst tensor size does not match copy atom size.");
```

Comment:
Why is this assert removed? If it is failing there is likely something wrong with your copy operation.